README.md

```diff
@@ -8,86 +8,51 @@ The Python library for the [Pocket Science Lab](https://pslab.io) from FOSSASIA.
+## Validate installation
 
-**Step 1**: Install the latest Python version on your computer and configure `PATH` variable to have both Python installation directory and the Scripts directory to access `pip` tools. In Windows, Python is installed in `C:` drive by default. We can set `$PATH` by opening the **Environment variables** dialog box by following the steps below:
-
-1. [Right click on My Computer] 
-2. Select "Properties"
-3. Open "System Properties"
-4. Click "Advanced" tab
-5. Click "Environment Variables" button
-6. Look for "**_PATH_**" in "System Variables" section and click on it and press "Edit" button
-7. To the end of "Variable value" text box, append "`C:\Python34\;C:\Python34\Scripts\;`" (without quotes and `34` may differ depending on the python version installed. It could be 35, 37 ...)
-8. Click "OK" twice to save and move out from path windows
-
-**Step 2**: Open up command prompt and execute the following commands to install the required dependencies.
-
-    $ pip install pyserial
-    $ pip install numpy
-
-#### Validate
-
-1. Download the PSLab-Python library from this repository and extract it to a directory.
-2. Browse in to that directory and create a new file named `test-pslab-libs.py`
-3. Paste the following code into that file and save it.
+1. Plug in the PSLab device and check that both the LEDs light up.
+2. The following piece of code should run without errors:
 ```
 from PSL import sciencelab
 I = sciencelab.connect()
 capacitance = I.get_capacitance()
 print(capacitance)
 ```
-4. Plug in the PSLab device and check if both the LEDs are turned on.
-5. Now run this file by typing `python test-pslab-libs.py` on a command prompt and observe a numerical value printed on the screen along with PSLab device version and the port it is connected to.
-
-### How to Setup the Development Environment
 
-To set up the development environment, install the packages mentioned in dependencies. For building GUI's you can use Qt Designer or create a frontend using any other compatible technology. The PSLab desktop app is using Electron for example.
-
-## How to Build the Documentation Website
+## Communication
 
-First install sphinx by running following command
+* If you encounter any bugs, please file them in our [issue tracker](https://github.com/fossasia/pslab-python/issues).
+* You can chat with the PSLab developers on [Gitter](https://gitter.im/fossasia/pslab).
+* There is also a [mailing list](https://groups.google.com/forum/#!forum/pslab-fossasia).
 
-    pip install -U Sphinx
+Wherever we interact, we strive to follow the [FOSSASIA Code of Conduct](https://fossasia.org/coc/).
 
-Then go to pslab/docs and run the following command
+## Contributing
 
-    $ make html
+See [doc/development.rst](https://github.com/fossasia/pslab-python/blob/development/docs/development.rst) to get started.
 
 ## License
 
-The library is free and open source software licensed under the [GPL v3](LICENSE). The copyright is owned by FOSSASIA. 
```

Comment:
\[GPL v3\](LICENSE) would link the current branch license file..

Reply:
The reason I changed it is that the current link will 404 on PyPI. Is it OK if I link with the full gtihub URL?

Reply:
Ahh I see. Yea I guess the full Git URL would do great :+1: